src/test/programs/federal/snap.test.js

```diff
@@ -2,18 +2,10 @@ import { getSNAPBenefits } from '../../../programs/federal/snap';
 
 // CLIENTS
 import { CLIENT_DEFAULTS } from '../../../utils/CLIENT_DEFAULTS';
+import { sampleClients } from '../../sampleClients';
 import { cloneDeep } from 'lodash';
 
 const defaultClient       = cloneDeep( CLIENT_DEFAULTS );
-const variant1            = cloneDeep( CLIENT_DEFAULTS );
-variant1.current.hasSnap  = true;
-// Head of household member is already there
-variant1.current.household.push( {"m_age":30,"m_role":"spouse","m_disabled":false} );
-variant1.current.household.push( {"m_age":12,"m_role":"member","m_disabled":false} );
-variant1.current.earned   = 2165;
-variant1.current.shelter  = "renter";
-variant1.current.rent     = 600;
-// SNAP never needs to test future
 
 describe('getSNAPBenefits', () => {
   describe('default client', () => {
```

Comment:
Didn't add the old test for the default client. Should come out to 0. That's not blocking, so we'll merge this, but maybe it can be in the next PR.